README.md

```diff
@@ -38,67 +30,56 @@ rustup target add wasm32-unknown-unknown --toolchain nightly
+### 2. Clone this workshop
 
-Clone your copy of the workshop codebase and switch to the `workshop` branch
+Clone your copy of the workshop codebase 
 
 ```zsh
-git clone https://github.com/<INSERT-YOUR-GITHUB-USERNAME>/utxo-workshop.git
-git fetch origin workshop:workshop
-git checkout workshop
+git clone https://github.com/substrate-developer-hub/utxo-workshop.git
+```
+
+## UI Demo
 
+In this UI demo, you will interact with the UTXO blockchain via the [Polkadot UI](https://substrate.dev/docs/en/development/front-end/polkadot-js). 
+
+The following demo takes you through a scenario where: 
+- Alice already owns a UTXO of value 100 upon genesis
+- Alice sends Bob a UTXO with value 50, tipping the remainder to validators
+
+1. Compile and build a release in dev mode
+```
 # Initialize your Wasm Build environment:
 ./scripts/init.sh
 
 # Build Wasm and native code:
 cargo build --release
 ```
 
-## Demo Polkadot UI
-
-You can use the [Polkadot UI](https://substrate.dev/docs/en/development/front-end/polkadot-js) to exercise the capabilities of your UTXO blockchain to ensure that everything is working as expected. First, you will need ensure that any existing UTXO blockchain data has been removed and then build the UTXO blockchain:
-
+2. Start your node & start producing blocks:
 ```zsh
-./target/release/utxo-workshop purge-chain --dev
 ./target/release/utxo-workshop --dev
-```
 
-You will notice that when you start the UTXO blockchain there is some output that is provided to help you with the following steps:
-```
-Initial UTXO Hash: 0x02f8ef6ff423559b1d07a31075d83c00883fbf02e054af102ab4a0ed8c50d2c5
-Transaction #1 TransactionOutput {
-	value: 100,
-	pubkey: 0x8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48,
-	salt: 0 },
-	Hash: 0x6cef4a2aa20b106a010cc0cdeebd7f48767372e9b68e37704d595c6abed6b2be
-Transaction #2 TransactionOutput {
-	value: 340282366920938463463374607431768211355,
-	pubkey: 0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d,
-	salt: 0 },
-	Hash: 0x4478cdec8efc46049b9b5ef6f615255455c39b67e9a932314dc81a2d63bd9681
+# If you already modified state, run this to purge the chain
+./target/release/utxo-workshop purge-chain --dev
 ```
 
-1. Visit [Polkadot JS](https://polkadot.js.org/apps/#/settings)
-  - Make sure you are using a browser that is compatible with local development or [consider running the UI locally](https://github.com/polkadot-js/apps)
+3. In the console, notice the helper printouts. In particular, notice the default account `Alice` was already has `100 UTXO` upon the genesis block.
+
+4. Open [Polkadot JS](https://polkadot.js.org/apps/#/settings), making sure the client is connected to your local node by going to Settings > General, and selecting `Local Node` in the `remote node` dropdown.
+
+5. **Declare the custom datatypes in PolkadotJS**, since the JS client cannot authomatically infer this from the UTXO module. Go to Settings > Developer tab and paste in the following JSON:
```

Comment:
```suggestion
5. **Declare the custom datatypes in PolkadotJS**, since the JS client cannot authomatically infer this from the UTXO pallet. Go to Settings > Developer tab and paste in the following JSON:
```